addons/block_code/ui/blocks/utilities/snap_point/snap_point.tscn

```diff
@@ -6,10 +6,9 @@
 anchors_preset = 15
 anchor_right = 1.0
 anchor_bottom = 1.0
-offset_right = -1152.0
-offset_bottom = -648.0
```

Comment:
This seems unrelated. Is it intentional?

Reply:
Yeah, not intentional. Not sure what that's about - it doesn't really look _right_ - but no reason to change it here. I removed that change now :)

Reply:
Maybe it's something that was set in the scene at some point but actually had no effect? 🤷